blueprints/foundational/config.ts

```diff
@@ -0,0 +1,293 @@
+import {
+  baseConfig,
+  Config,
+  GLOBAL_REGION,
+  Template,
+} from '@superluminar-io/aws-luminarlz-cli/lib/config';
+
+/**
+ * The deployed version of the AWS Landing zone accelerator.
+ */
+export const AWS_ACCELERATOR_VERSION = '1.12.1';
+
+/**
+ * The AWS management account, organization and root organizational unit used in this project.
```

Comment:
```suggestion
 * The AWS management account, organization and root organizational unit used in this project.
 * TODO: update with your data
```

Reply:
This is automatically filled in during the initialization step.